Terminal.Gui/View/Frame.cs

```diff
@@ -198,7 +198,9 @@ public override void OnDrawContent (Rect contentArea)
 
 			}
 
-			if (Id == "Border" && canDrawBorder && Thickness.Top > 0 && maxTitleWidth > 0 && !string.IsNullOrEmpty (Parent?.Title)) {
+			if (Id == "Border" && canDrawBorder && Thickness.Top > 0 && maxTitleWidth > 0 && !string.IsNullOrEmpty (Parent?.Title)
```

Comment:
I find it curious that you are continuing to add to this code, which I clearly stated was a prototype, instead of implementing Border, Padding, Margin correctly (each as a subclass of Frame)? 

All of this code (my old junk and your new tweaking of that junk) will need to be refactored once there are 3 Frame-based classes, and when Border contains a set of subviews that actually do the drawing vs. using LineCanvas directly.

Reply:
I only wanted to get a minimum border for the `TabView` here and was the solution I found.